test/helpers/data/staging-cms-helper.js

```diff
@@ -0,0 +1,22 @@
+const knexConfig = require('../../../knexfile').staging
+const knex = require('knex')(knexConfig)
+var Promise = require('bluebird').Promise
+
+var inserts = []
+
+module.exports.insertDependencies = function (cmsRecords) {
+  return knex('cms')
+    .insert(cmsRecords)
+    .returning('id')
```

Comment:
Will this not return `{id: the_id}`? So the insert below would need to use `id: id.id`?

Reply:
I've checked this out, knex is smart enough to just return the thing, but it returns it in a array ie `[theid]`
I've updated to retrieve just `theid` rather than the whole array